ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactScrollView.java

```diff
@@ -21,10 +21,12 @@
 import android.util.Log;
 import android.view.MotionEvent;
 import android.view.View;
+import android.view.ViewGroup;
 import android.widget.OverScroller;
 import android.widget.ScrollView;
 
 import com.facebook.react.bridge.ReactContext;
+import com.facebook.common.logging.FLog;
```

Comment:
If I recall correctly you can build UIExplorer with buck by running `buck build uiexplorer`